salt/runner.py

```diff
@@ -156,6 +157,12 @@ def run(self):
                                            async_pub['jid'],
                                            False,  # Don't daemonize
                                            )
+                msg = 'Runner completed'
```

Comment:
Before, we didnt return the stuff in the job cache dir, now that we have it in, for users to retrieve it, they need the jid.